graylog2-server/src/main/java/org/graylog/security/shares/DefaultGranteeService.java

```diff
@@ -68,6 +76,11 @@ public DefaultGranteeService(UserService userService, GRNRegistry grnRegistry, G
     }
 
     private ImmutableSet<AvailableGrantee> getAvailableUserGrantees(User sharingUser) {
+        final Optional<UserAndTeamsConfig> config = Optional.ofNullable(clusterConfigService.get(UserAndTeamsConfig.class));
```

Comment:
I'd reccomend using 
`clusterConfigService.getOrDefault(UserAndTeamsConfig.class, UserAndTeamsConfig.create(true, true));`

or even better create a constant:
`clusterConfigService.getOrDefault(UserAndTeamsConfig.class, UserAndTeamsConfig.DEFAULT_VALUES);`

Reply:
We are initializing in the frontend, when the form is opened. But it's a good idea to have defined default behavior in the backend.